### DISCUSSION TITLE
Clarification on whether SES/S3 mandatory

### DISCUSSION THREAD

**Discussion by lowlyocean** [chaskiq/chaskiq#532] [Q&A]
Hi, was trying to deploy to Heroku and noticed Amazon S3 and SES parameters are required. I started to look at satisfying the requirement but I'm wondering - why is this really necessary? Can it be omitted, and if so, what would be the price to pay?

All I'm looking for is messenger widget to interact with customers, so not quite clear to me why any email or storage integration is mandatory.

**Comment by michelson** [comment]
Hi Daniel, thanks for asking, 
Storage is mandatory in order to be able to upload/send files on conversations. The Email -> SES is not strictly mandatory, but that service enables the continuation of conversations through email, so it's very useful, right now Amazon SES is the only implemented flavor for this, but eventually, other email receiver processors could be implemented.
Note that you could use normal SMTP credentials without SES, but you will lose the incoming messages, (when the visitor replies from their email)
By the way, Amazon SES is the cheapest email service, if you use one.
